accounts/forms.py

```diff
@@ -13,6 +14,12 @@ class Meta:
         model = CustomUser
         fields = ['username', 'name', 'email', 'password1', 'password2', 'bio', 'github_link', 'avatar']
 
+<<<<<<< HEAD
```

Comment:
It looks like there was some sort of merge error here where you've got the git messages mixed in with your code.